src/core/actions.py

```diff
@@ -0,0 +1,26 @@
+from inspect import getmembers, isfunction, ismodule
+
+from src.jbi import process as jbi_process
+
+module_dict = {jbi_process.ACTION_KEY: jbi_process.actions}
+
+
+def get_action_context_by_key(key):
+    if key not in module_dict:
+        assert False, "Unknown key requested"
+    return get_action_context_from_module(action_module=module_dict.get(module_dict))
+
+
+def get_action_context_from_module(action_module):
```

Comment:
`action_context` is a dictionary used to connect the JSON "action" chosen via config; the dictionary is keyed with method_name and values of python `function`'s